tiledb/sm/c_api/tiledb.h

```diff
@@ -2372,7 +2372,8 @@ TILEDB_EXPORT int32_t tiledb_array_schema_set_domain(
  *
  * @param ctx The TileDB context.
  * @param array_schema The array schema.
- * @param capacity The capacity to be set.
+ * @param capacity The capacity to be set. For dense arrays, tile capacity is
```

Comment:
`Capacity` always refers to the number of cells in a **sparse tile**, present in sparse fragments. Note that you can create sparse fragments on dense arrays as well, by writing random cells to it (i.e., not contiguous in a hyper-rectangle).

Reply:
How about `For dense arrays, *tile capacity* controls the maximum number of cells per data tile when writing a (sparse) [fragment](tutorials/fragments-consolidation.html).`

Reply:
Counter suggestion:

```
@param capacity The capacity of a sparse data tile. Note that 
    sparse data tiles exist in sparse fragments, which can be created
    in both sparse and dense arrays. For more details, 
    see https://docs.tiledb.io/en/stable/tutorials/tiling-sparse.html
```

I would avoid adding a title to the docs link, in case we change it and we forget about the docstring.

Reply:
Updated. 

> I would avoid adding a title to the docs link, in case we change it and we forget about the docstring.

I was hoping to get proper linking behavior when reading the `stable` vs `latest` vs versioned doc builds (whereas an absolute link will always go to that version).

Reply:
You can try a local build of the docs to see if linking works. I suspect it won't, as the doxygen source is unrelated to the rest of the RST doc structure.

Reply:
Yes, it does work correctly with a local build, because the API docs are at the root of the tree.
```
_build/html/c++-api.html
_build/html/tutorials/tiling-sparse.html
```

Reply:
btw, I haven't tried it, but breathe does support inline RST (in case it comes in handy someday):

https://breathe.readthedocs.io/en/latest/markups.html#supported-markups